public/js/main.js

```diff
@@ -124,12 +124,12 @@ function completeTask(taskId) {
   });
 
   saveData(todoList);
-  render();
+  renderData();
 }
 
 listData.addEventListener("click", (event) => {
-  let element = event.target;
-  let value = element.getAttribute("class");
+  const element = event.target;
+  const value = element.getAttribute("class");
```

Comment:
The variable should describe their domain which is pointing.
Eg: `value` -> `todoListClasses`
- [ ] refactor